pennylane/devices/tests/test_gates.py

```diff
@@ -188,13 +196,20 @@
     (qml.RZ, rz),
 ]
 # list of all non-parametrized two-qubit gates
-two_qubit = [(qml.CNOT, CNOT), (qml.SWAP, SWAP), (qml.ISWAP, ISWAP), (qml.CZ, CZ), (qml.CY, CY)]
+two_qubit = [
```

Comment:
Hi @paniash, how come such formatting changes were applied? We'd usually be formatting files with `black -l 100 pennylane` where[ `pennylane` is the subdirectory in the repo](https://github.com/PennyLaneAI/pennylane/tree/master/pennylane). Same for the `tests` subdirectory.

Reply:
@antalszava I applied the formatting as suggested. Not sure what happened. However, I ran it once again on the subdirectory `pennylane` and `tests` and got the following output:

```
$ black -l 100 tests pennylane
All done! ✨ 🍰 ✨
282 files left unchanged.
```

Reply:
Got it! Which version of `black` was it? Our check is using `black-21.6b0`. It would be great to revert the formatting changes so that only the IsingYY addition is contained in this PR.

Reply:
I'm on version `21.6b0`, so no version incompatibility.